bootstrap.sh

```diff
@@ -57,7 +64,13 @@ while [ -n "$1" ]; do
     esac
 done
 
-pyver=$($python -V | sed -n 's/Python \([0-9]\+\)\.\([0-9]\+\)\..*/\1.\2/p')
+pyver=$($python -V 2>&1 | sed -n 's/Python \([0-9]\+\)\.\([0-9]\+\)\..*/\1.\2/p')
+pymajver=$(echo $pyver | cut -f 1 -d '.')
+pyminver=$(echo $pyver | cut -f 2 -d '.')
+
+if [ $pymajver -lt 3 ] || [ $pyminver -lt 6 ]; then
+    PYTHON_REQ $pyver
+fi
```

Comment:
I don't like that we have to redo some of the logic of the `reframe` module in Bash. We could simply run `./bin/reframe -V` and check if it fails. In any case, `./bootstrap.sh` is meant to be executed from the top-level resource directory.

Reply:
We could have something like: `echo "Bootstrapping ReFrame $(./bin/reframe -V) ..."`

Reply:
The problem with this is that it won't respect a particular python passed as a cli option to the bootstrap script.

Reply:
Ok, but the problem is that this is not too portable. This `sed` command, for example, is not valid under OSX. You need GNU sed for this. I would suggest doing the check in a smart way directly through Python:

```bash
# NOTE: We need to exit with a zero code if the Python version is the correct one, so we invert the comparison
if python -c 'import sys; sys.exit(sys.version_info[:2] > (3, 6))'; then
    echo -e "ReFrame requires Python >= 3.6"
fi
```